.github/workflows/release.yml

```diff
@@ -21,9 +21,9 @@ on:
         required: false
         default: 'false'
       versionParent:
-        description: 'Specify Che Parent version here, if *releaseParent is true'
+        description: 'Specify Che Parent version here'
```

Comment:
Does `releaseParent` even work now that we have disabled pushing to Nexus?

Reply:
we still have the functionality to release Che Parent (should we ever need to do that for any reason), and this performs a release - tagging & building mvn artifact in a local repo, which would then be needed to build new che-server that would depend on that